src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Release Notes
+description: .
+tree_item_index: 700
+review:
+  comment: ''
+  date: '2020-03-02'
+  status: ok
+toc: true
+---
+
+Welcome to the Release Notes for **Nuxeo Drive 4.5.0**
+
+## Breaking Changes
+
+### Direct Transfer
+
+The Direct Transfer feature has been disabled for the moment; the implementation is currently being reworked to be more stable.  
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2040](https://jira.nuxeo.com/browse/NXDRIVE-2040)
+
+## Improvements
+
+### S3 Direct Upload
+
+When the Amazon S3 addon is installed and configured, Nuxeo Drive will automatically use it when uploading files to Nuxeo. It's a great improvement as it will lower the work on the Nuxeo side and leverage high-speed servers from Amazon.
```

Comment:
It's is great improvement as it will lower the load on your Nuxeo server and users will benefit from high-speed transfer capabilities from Amazon.

Reply:
Also:  "When the Amazon S3 add-on is installed and configured for Direct Upload"